client/web/src/hooks/auth.ts

```diff
@@ -28,11 +28,10 @@ export default function useAuth() {
 
   const loadAuth = useCallback(() => {
     setLoading(true)
-    return apiFetch("/auth", "GET")
-      .then((r) => r.json())
+    return apiFetch<AuthResponse>("/auth", "GET")
```

Comment:
These changes throughout add typing to the fetchers, so that we get a typed response.